### ISSUE TITLE
data 

### ISSUE THREAD

**Issue by lclszsdnr** [VICO-UoE/CIN-SSL#2]
Hello dear author, thank you very much for your contribution. May I ask if the training set provided corresponds to the real label data in the paper or a combination of real and fake note data？
